iacrcc/iacrcc.cls

```diff
@@ -359,48 +351,24 @@ xscale=\@OrigHeightRecip*\@curXheight,transform shape]
   % Also check if the font (\familydefault) or the margings (\paperwidth) were modified.
   % Throw an error if so, these are needed for meta-data collection
   \ifx\store@addauthor\addauthor\else
-    \ClassError{iacrcc}{%
-      ^^J%
-      The \string\addauthor\space macro seems to be redefined%
-      ^^J%
-    }{}%
+    \ClassError{iacrcc}{The \string\addauthor\space macro seems to be redefined}{}%
```

Comment:
I'm curious why you removed the ^^J in the class errors. It causes them to be more visible.

Reply:
I can explain in more detail in our next call: the leading spaces caused some testcases to throw errors.